YourToDo/templates/planner/ajax_events_in_planner.html

```diff
@@ -1,69 +1,116 @@
 <div class="planner-week-content">
-            <table class="planner-container">
-                <!-- Column Specification: Left sidebar, 7 days -->
-                <col class="planner-week-left-sidebar">
-                <col class="planner-week-days-day">
-                <col class="planner-week-days-day">
-                <col class="planner-week-days-day">
-                <col class="planner-week-days-day">
-                <col class="planner-week-days-day">
-                <col class="planner-week-days-day">
-                <col class="planner-week-days-day">
-{% for category in categoriesInPlanner %}
-<tr class="planner-week-row">
-    <td class="planner-week-category-container planner-category-color-{{ category.get_color }}">
-        {{ category.get_name }}
-    </td>
-    <td>
-        {% for event in eventsInViewStartDate %}
-            {% if event.get_parentCategory == category %}
-                {{ event.get_name }}
-            {% endif %}
+    <table class="planner-container planner-container-content">
+        <!-- Column Specification: Left sidebar, 7 days -->
+        <col class="planner-week-left-sidebar">
+        <col class="planner-week-days-day">
+        <col class="planner-week-days-day">
+        <col class="planner-week-days-day">
+        <col class="planner-week-days-day">
+        <col class="planner-week-days-day">
+        <col class="planner-week-days-day">
+        <col class="planner-week-days-day">
+        {% for category in categoriesInPlanner %}
+        <tr class="planner-week-row">
+            <td class="planner-week-category-container planner-category-color-{{ category.get_color }}">
+                {{ category.get_name }}
+            </td>
+            <td class="planner-week-event-block">
+                <table class="planner-week-event-container">
+                    {% for event in eventsInViewStartDate %}
+                        {% if event.get_parentCategory == category %}
+                        <tr>
+                            <td class="planner-category-color-{{ category.get_color }}">
+                                <div>
+                                    {{ event.get_name }}
+                                </div>
+                            </td>
+                        </tr>
+                        {% endif %}
+                    {% endfor %}
+                    <tr>
```

Comment:
These extra rows are for debugging. I'm leaving them in here until we're completely finished with linking events to their respective modals.